vowpalwabbit/reductions/cb/cb_to_cb_adf.cc

```diff
@@ -194,22 +209,25 @@ VW::LEARNER::base_learner* cb_to_cb_adf_setup(VW::setup_base_i& stack_builder)
 
   // see csoaa.cc ~ line 894 / setup for csldf_setup
   all.example_parser->emptylines_separate_examples = false;
+  VW::prediction_type_t out_pred_type;
 
   if (data->explore_mode)
   {
     data->adf_learner = as_multiline(base->get_learner_by_name_prefix("cb_explore_adf_"));
+    out_pred_type = base->get_output_prediction_type();
```

Comment:
We're still not able to determine action_scores vs action_probs here. The `finish_example` does something and it probably depends on being either scores or probs right?